omod-1.9/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_9/PatientIdentifierController1_9Test.java

```diff
@@ -125,6 +126,20 @@ public void shouldVoidAPatientIdentifier() throws Exception {
 	}
 	
 	@Test
+	public void shouldListAllPatientIdentifiersWithVoidedIdentifiersForAPatient() throws Exception {
+		assertEquals(false, service.getPatientIdentifierByUuid(getUuid()).isVoided());
+		MockHttpServletRequest req = request(RequestMethod.DELETE, getURI() + "/" + getUuid());
+		req.addParameter("!purge", "");
+		final String reason = "none";
+		req.addParameter("reason", reason);
+		handle(req);
+		assertEquals(true, service.getPatientIdentifierByUuid(getUuid()).isVoided());
+		SimpleObject result = deserialize(handle(newGetRequest(getURI(), new Parameter(
+			RestConstants.REQUEST_PROPERTY_FOR_INCLUDE_ALL, "true"))));
+		assertNotEquals(getAllCount(), Util.getResultsSize(result));
```

Comment:
Can you make another call and corresponding assertion where the includeAll parameter has a value of false?